openupgrade_scripts/scripts/website/15.0.1.0/post-migration.py

```diff
@@ -36,6 +36,7 @@ def extract_footer_copyright_company_name(env):
             or f"Copyright © {website.company_id.name}",
         )
         main_copyright_view.with_context(website_id=website.id).arch_db = new_arch
+        view.unlink()
```

Comment:
Probably you won't be able to do this due to inheritance chain

Reply:
Right, with this additional code the inheritances will also be eliminated, thank you!